using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -1,18 +1,21 @@
 # Identity API reference
 
-## Sync identifiers
+## Sync identifiers<a id="syncIdentifiersTitle"></a>
 
-Updates the specified customer ID with the Adobe Experience Cloud ID service.
+The  `syncIdentifier()` and `syncIdentifiers()` APIs update the specified customer IDs  with the Adobe Experience Cloud ID service. 
```

Comment:
are there extra spaces between "The" and "`syncIdentifier()`" and also between "IDs" and "with the Adobe..."? If yes, please remove.